README.md

```diff
@@ -25,3 +25,41 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+
+Runtime Analysis of the Divide and Conquer Sort Algorithm:
+
+The final runtime complexity of the implemented divide and conquer sorting algorithm is:
+T(n) = 3T(n/3) + (n)
+T(n) = 3(3T(n/9) + (n/3)) + (n)
+T(n) = 3^2(T)(n/9) + 4(n)
+T(n) = 9(3T(n/27) + n/9) + 4(n) 
+T(n) = 27(T)(n/27) + 13n
+....
+T(n) = 3^i (T)(n/3^i) + (3^i - 1)n/2
+
+n/3^i = 1  => i = log3(n)
+
+T(n) = 3^log3(n) T(1) + nlog3(n)
+T(n) = n * c + nlog3(n)  (c representing O(1). the constant amount of work)
```

Comment:
Please have a look at what we did in class. You can't just introduce an arbitrary c halfway through.

Reply:
I thought it was okay, because we get it from T(1) which is a base case where it is explicitly constant? I tried looking through the notes and I have not found anything like this yet.